content/outcomes/cloud-native-development/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cloud Native Development
+weight: -1
+layout: single
+description: "Learn the basics of cloud native development with Spring. This developer pre-engagement guide helps you to get the most of an engagement following the Tanzu Labs methodology."
+team:
+ - VMware Tanzu Labs
+---
+ 
+## Why is this learning path important?
+
+This learning path provides the foundational knowledge around modern, cloud-native software development with Spring and Cloud-Foundry. It has been conceived as pre-work to get the most of an application-related software development engagement with Tanzu Labs. 
+
+## Who is this learning path for?
+ 
+This learning path is for software engineers and other members of the technical development team involved in a Tanzu Labs engagement.
+
+## What you will learn
+ 
+Each section of this guide provides details about the skills or technologies in question, and includes links to specific and targeted information to assist you. Each section is followed by suggested homework assignments and additional resources.
+
+<<<<<<< HEAD
+It should take about eight hours to complete the material in this learning path. However, the more time you put into the learnng path, the more you will get out of the engagement.
+ 
+<<<<<<< HEAD
+This learning path includes the following topics:
+=======
+Going through the material of this learning path should not take more than 8 hours. The more time you spend, for example by studying deeper the provided additional resources, the more you will get out of this learning path.
+>>>>>>> c756708ac15d36a6d1c50b284f27ec4c49ffb4ed
+ 
+**General skills**
+=======
+The following topics are covered:
+
+**Development ecosystem**
+>>>>>>> 244d986b9dee0e461cd063af7babd7b86e487dff
```

Comment:
More conflicts.

Reply:
Weird. I'll take a look. I did just hit the "update branch" button but it usually prevents merge conflicts.

Reply:
Fixed in 961ee163a68737d1eb20b82c27aaa8537cdb402e !